src/test/java/domain/AttendanceHistoryLoaderTest.java

```diff
@@ -0,0 +1,50 @@
+package domain;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.jupiter.api.Test;
+
+public class AttendanceHistoryLoaderTest {
+
+    private File createTempCsvFile(String csvData) throws IOException {
+        Path tempFile = Files.createTempFile("test_attendance", ".csv");
+        Files.write(tempFile, csvData.getBytes());
+
+        return tempFile.toFile();
+    }
+
+    @Test
+    void 유효한_형식의_파일에_대해_Crew를_정상적으로_초기화한다() throws Exception {
+        String csvData = "nickname,datetime\n에드,2025-02-13 08:30\n제프,2025-02-14 09:15\n";
```

Comment:
```suggestion
        String csvData = """
                nickname,datetime
                에드,2025-02-13 08:30
                제프,2025-02-14 09:15
                """;
```
텍스트 블럭을 활용하면 좋아요~
https://www.baeldung.com/java-text-blocks

Reply:
감사합니다! 적용 완료 하였습니다